src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -94,6 +94,7 @@
                     + "                                         ERROR,WARN,INFO,DEBUG,TRACE [default: INFO].\n"
                     + " -Q, --quickcheck-test-mode              Run in QuickCheck test mode\n"
                     + " -P <seconds>, --cm-poll-interval        Configuration manager poll interval [default: 1]\n"
+                    + " -v, --verify                            Compute and verify checksums for log entries.\n"
```

Comment:
Well, I propose this:

When checksum is disabled, 0xFF is written to the version field, which tells the implementation to just read the checksum field and make sure that they are all 0s. I think that the chance of 0xFF written to the version field AND the field being zero'd is about as likely as a checksum collision.